railties/lib/rails/generators/rails/script/USAGE

```diff
@@ -0,0 +1,18 @@
+Description:
+    Generate a one-off or general purpose script, such as a data migration
+    script, cleanup script, etc.
+
+Example:
+    `bin/rails generate script my_script`
+
+    This will create:
+        script/my_script.rb
+
+    You can run the script using:
+        `ruby script/my_script.rb`
```

Comment:
Could it be valuable to encourage users to execute the script with the [rails runner](https://edgeguides.rubyonrails.org/command_line.html#bin-rails-runner) command so that the script itself has access to Rails classes (like the ones in `app/models`)?

Reply:
While the generated `require_relative "../config/environment"` does the trick and allows the script to be run using just `ruby`, it adds friction to moving the script around. Using `rails runner` would do that for free.

Reply:
@seanpdoyle As Neil said, if your script uses `require_relative "../config/environment"` (which the script generator adds for you), you can run it with `ruby` and it will work the same as if you ran it with `rails runner`.

If you prefer not to use `require_relative "../config/environment"`, you can create your scripts manually without it and run them with `rails runner`.

Reply:
> @seanpdoyle The generator creates a script that has `require_relative "../config/environment"`, so you can run the script with `ruby` and it will work the same as if you ran it with `rails runner`.

Thanks for clarifying @jeromedalbert, I wasn't clear until I read your comment and tried the generator whether the scripts were living in Ruby or in my Rails project